tests/test_MathVector.py

```diff
@@ -88,4 +88,21 @@ def test_angleAxisRotationV():
 
     SOL = angleAxisRotationV(POS,ANG,AXIS,ANCHOR)
     for pos,ang,ax,anch,sol in zip(POS,ANG,AXIS,ANCHOR,SOL):
-        assert np.amax(np.abs(angleAxisRotation(pos,ang,ax,anch)-sol))<1e-10, "bad angleAxisRotationV"
\ No newline at end of file
+        assert np.amax(np.abs(angleAxisRotation(pos,ang,ax,anch)-sol))<1e-10, "bad angleAxisRotationV"
+
+
+
+def test_ellipticV():
+    #random input
+    INP = np.random.rand(1000,4)
```

Comment:
If this test ever fails because of a certain combination, it will be hard to recover the problematic input.
We could use a pseudo number generator with a fixed seed instead.

```python 


seed = 6969
np.random.seed(seed)
test_input = np.random.rand(1000,4)


# print(test_input) 
# Should always return:
# [[0.90778746 0.02226607 0.42943218 0.30438451]
#  [0.6351356  0.53867276 0.74523766 0.88416302]
#  [0.96144443 0.50252932 0.74019615 0.16491463]
#  ...
#  [0.44344936 0.48382863 0.24196537 0.83127193]
#  [0.16418535 0.45008313 0.48605127 0.92810509]
#  [0.09808894 0.20805508 0.73558164 0.71283222]]
```